docs/source/notes/windows.rst

```diff
@@ -0,0 +1,258 @@
+Finally, you can start your compiling process.
+
+Installation
+------------
+
+Package not found in win-32 channel.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bat
+
+    Solving environment: failed
+
+    PackagesNotFoundError: The following packages are not available from current channels:
+
+    - pytorch
+
+    Current channels:
+    - https://conda.anaconda.org/pytorch/win-32
+    - https://conda.anaconda.org/pytorch/noarch
+    - https://repo.continuum.io/pkgs/main/win-32
+    - https://repo.continuum.io/pkgs/main/noarch
+    - https://repo.continuum.io/pkgs/free/win-32
+    - https://repo.continuum.io/pkgs/free/noarch
+    - https://repo.continuum.io/pkgs/r/win-32
+    - https://repo.continuum.io/pkgs/r/noarch
+    - https://repo.continuum.io/pkgs/pro/win-32
+    - https://repo.continuum.io/pkgs/pro/noarch
+    - https://repo.continuum.io/pkgs/msys2/win-32
+    - https://repo.continuum.io/pkgs/msys2/noarch
+
+PyTorch doesn't work on 32-bit system. Please use Windows and
+Python 64-bit version.
+
+Why are there no Python 2 packages for Windows?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Because it's not stable enough. There're some issues that need to
+be solved before we officially release it. You can build it by yourself.
+
+Import error
+^^^^^^^^^^^^
+
+.. code-block:: py3tb
+
+    from torch._C import *
+
+    ImportError: DLL load failed: The specified module could not be found.
+
+
+The problem is caused by the missing of the essential files. Actually,
+we include almost all the essential files that PyTorch need except VC2017
+redistributable. You can resolve this by typing the following command.
+
+.. code-block:: bat
+
+    conda install -c peterjc123 vc vs2017_runtime
+
+Another possible cause may be you are using GPU version without NVIDIA
+graphics cards. Please replace your GPU package with the CPU one.
+
+Usage (multiprocessing)
+-------------------------------------------------------
+
+Multiprocessing error without if-clause protection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: py3tb
+
+    RuntimeError:
+   	An attempt has been made to start a new process before the
+   	current process has finished its bootstrapping phase.
+
+       This probably means that you are not using fork to start your
+       child processes and you have forgotten to use the proper idiom
+       in the main module:
+
+           if __name__ == '__main__':
+               freeze_support()
+               ...
+
+       The "freeze_support()" line can be omitted if the program
+       is not going to be frozen to produce an executable.
+
+The implementation of ``multiprocessing`` is different on Windows, which
+uses ``spawn`` instead of ``fork``. So we have to wrap the code with an
+if-clause to protect the code from executing multiple times. Refactor
+your code into the following structure.
+
+.. code-block:: python
+
+    import torch
+
+    def main()
+        for i, (x, y) in dataloader:
```

Comment:
nit: you need `enumerate` to get `i`. Also can we just say `data` instead of `(x, y)`